storage/xtradb/btr/btr0sea.cc

```diff
@@ -60,26 +60,19 @@ UNIV_INTERN ulint		btr_search_n_succ	= 0;
 UNIV_INTERN ulint		btr_search_n_hash_fail	= 0;
 #endif /* UNIV_SEARCH_PERF_STAT */
 
-/** padding to prevent other memory update
-hotspots from residing on the same memory
-cache line as btr_search_latch */
-UNIV_INTERN byte		btr_sea_pad1[64];
-
 /** Array of latches protecting individual AHI partitions. The latches
 protect: (1) positions of records on those pages where a hash index from the
 corresponding AHI partition has been built.
 NOTE: They do not protect values of non-ordering fields within a record from
 being updated in-place! We can use fact (1) to perform unique searches to
 indexes. */
 
-UNIV_INTERN prio_rw_lock_t*	btr_search_latch_arr;
-
-/** padding to prevent other memory update hotspots from residing on
-the same memory cache line */
-UNIV_INTERN byte		btr_sea_pad2[64];
+UNIV_INTERN prio_rw_lock_t*	btr_search_latch_arr
+					MY_ALIGNED(CPU_LEVEL1_DCACHE_LINESIZE);
 
 /** The adaptive hash index */
-UNIV_INTERN btr_search_sys_t*	btr_search_sys;
+UNIV_INTERN btr_search_sys_t*	btr_search_sys
+					MY_ALIGNED(CPU_LEVEL1_DCACHE_LINESIZE);
```

Comment:
Why this and only this file is aligned on CPU_LEVEL1_DCACHE_LINESIZE and not on CACHE_LINE_SIZE?